backend.native/tests/runtime/workers/worker_threadlocal_no_leak.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+import kotlin.native.concurrent.*
+
+@ThreadLocal
+var x = Any()
+
+fun main() {
+    val worker = Worker.start()
+
+    worker.execute(TransferMode.SAFE, {}) {
+        println(x)  // Make sure x is initialized
+    }
```

Comment:
Shouldn't we get the result to dispose the future here?
How does this work otherwise?

Reply:
(I guess `Unit` just doesn't "leak")

Reply:
I step in it every time… Fixed, thanks!